### PR TITLE
Laminas update

### PR DESCRIPTION
**Pull request by ewilde-imperial**
Updating from ZF3 to Laminas using the laminas-migration tool.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-    "name": "stroker/cache",
-    "description": "Provides a full page cache solution for ZF2",
+    "name": "ewilde-imperial/strokercache",
```
Comment (bramstroker) [inline composer.json]: Package name should still stay `stroker/cache`. I'll issue a new major version bump of this package when it is merged in master.

File: composer.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+    "description": "Provides a full page cache solution for Laminas",
     "type": "library",
     "keywords": [
-        "zf2",
+        "laminas",
         "cache",
         "fpc"
     ],
-    "homepage": "https://github.com/bramstroker/zf2-fullpage-cache",
+    "homepage": "https://github.com/ewilde-imperial/zf2-fullpage-cache",
```
Comment (bramstroker) [inline composer.json]: Same here

**Review by bramstroker — COMMENTED** [review]
Thanks for your efforts. PR looking good excect the package names in the composer.json.
If you can fix those I'm happy to merge.

File: composer.json (hunk 1 above)
Comment (ewilde-imperial) [reply composer.json]: Sorry, I will fix those...wasn't expecting such a quick response :)

Comment (bramstroker) [reply composer.json]: 👍

**Comment by bramstroker** [comment]
@ewilde-imperial Can you test with the `dev-master` in your project please? If everything works as expected I will release a new version.

**Comment by ewilde-imperial** [comment]
Absolutely, I will let you know

**Comment by ewilde-imperial** [comment]
btw, thanks for your work on this project - I have been using it for many years 👍

**Comment by bramstroker** [comment]
Nice to hear, glad you like it.

**Comment by ewilde-imperial** [comment]
I have tested this is working as expected, it is not in production yet though.

Specifically we are using it with the file and wincache adapters.

**Pull request merged**